test/utils.tests.js

```diff
@@ -0,0 +1,45 @@
+const { File, processFileController } = require('../utils.js');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+
+const expect = chai.expect;
+chai.use(chaiAsPromised);
+
+const testFilePath = `${__dirname}/testFile.json`;
+const invalidFilePath = './invalidPath';
+
+describe('File Class', () => {
+  const newFile = new File(testFilePath);
+
+  it('it should be an object that containts a filename property', () => {
```

Comment:
This is an unneeded test, you're testing if the behavior of an `Object`, rather you should tests if the property has the value you're expecting.